handlers/s3client.go

```diff
@@ -20,7 +22,7 @@ func CreateNewAwsClient() (*AwsClient, error) {
 	client := new(AwsClient)
 
 	sess, err := session.NewSession(&aws.Config{
-		Region: aws.String(os.Getenv("AWS_REGION")),
+		Region: aws.String("us-east-1"),
```

Comment:
try to get AWS_REGION from environment and use this for default?

Reply:
Switch back to env